src/App/components/InputGroup/index.js

```diff
@@ -51,7 +50,7 @@ const InputGroup = ({
         defaultValue: defaultValue || "",
         disabled: disabled || null,
         readOnly: readOnly || null,
-        autoComplete: autoComplete || null,
+        autocomplete: "on" || null,
```

Comment:
**NITPICK INC**
Our "Forms" documentation passes values to the `autoComplete` attribute (which should be renamed to `autocomplete`). Values such as `phone`, `given-name`, `username` is no longer supported with this approach.